app/controllers/journeys/income/IncomeCYAController.scala

```diff
@@ -76,9 +84,37 @@ class IncomeCYAController @Inject() (override val messagesApi: MessagesApi,
   def onSubmit(taxYear: TaxYear, businessId: BusinessId): Action[AnyContent] = (identify andThen getUserAnswers andThen requireData).async {
     implicit request =>
       val context = JourneyContextWithNino(taxYear, request.nino, businessId, request.mtditid, Income)
-      val result  = service.submitAnswers[IncomeJourneyAnswers](context, request.userAnswers)
+      val result = for {
+        updatedUserAnswers <- maybeClearSimplifiedExpensesTailoringAnswers(context)
+        _                  <- service.submitAnswers[IncomeJourneyAnswers](context, updatedUserAnswers)
+      } yield Redirect(journeys.routes.SectionCompletedStateController.onPageLoad(taxYear, businessId, Income, NormalMode))
+      handleResultT(result)
+  }
+
+  private def maybeClearSimplifiedExpensesTailoringAnswers(
+      ctx: JourneyContextWithNino)(implicit request: DataRequest[_], hc: HeaderCarrier, ec: ExecutionContext): ApiResultT[UserAnswers] = {
+    def clearSimplifiedExpensesIfTurnoverChangedToOverThreshold(previousIncomeWasUnderThreshold: Boolean,
+                                                                currentIncomeIsOverThreshold: Boolean,
+                                                                expensesTailoringIsSimplified: Boolean): ApiResultT[UserAnswers] =
+      if (previousIncomeWasUnderThreshold && currentIncomeIsOverThreshold && expensesTailoringIsSimplified) {
+        service.clearSimplifiedExpensesData(ctx)
+      } else {
+        EitherT.rightT(request.userAnswers)
+      }
 
-      handleSubmitAnswersResult(context, result)
+    for {
+      existingTotalIncomeAmount    <- service.getTotalIncome(ctx)
+      currentAnswersTurnoverAmount <- EitherT.fromEither[Future](request.valueOrNotFoundError(TurnoverIncomeAmountPage, ctx.businessId))
+      currentAnswersOtherIncomeAmount = request.getValue[BigDecimal](OtherIncomeAmountPage, ctx.businessId).getOrElse(BigDecimal(0))
+      currentAnswersTotalIncomeAmount = currentAnswersTurnoverAmount + currentAnswersOtherIncomeAmount
+      existingIncomeIsOverThreshold   = totalIncomeIsOverIncomeThreshold(existingTotalIncomeAmount)
+      currentIncomeIsOverThreshold    = totalIncomeIsOverIncomeThreshold(currentAnswersTotalIncomeAmount)
+      expensesTailoringIsSimplified   = request.getValue(ExpensesCategoriesPage, ctx.businessId).exists(_ != IndividualCategories)
```

Comment:
This line uses an `.exists()` because if there are no expenses then there's no point trying to clear expenses from BE.
Will only be true if Expenses Categories == TotalAmount | NoExpenses